core_backend/Makefile

```diff
@@ -78,15 +78,15 @@ teardown-dbs: stop-relational-db stop-vector-db
-	curl -X POST -d '{"content_text":"The tennis racquet, an extension of a players arm, is a marvel of engineering and design. From its early wooden iterations to todays advanced composite materials, the racquet has evolved to enhance performance on the court. With the right balance, weight, and string tension, a tennis racquet can make all the difference in those match-winning shots."}' -H 'Content-Type: application/json' $(ENDPOINT_URL)/content/create
-	curl -X POST -d '{"content_text":"From the aromatic scent of freshly ground beans to the invigorating first sip in the morning, coffee is more than just a beverage—its an experience. Originating from ancient Ethiopian plateaus, coffee has traveled the world, with each region offering its unique blend and roast. Whether youre a latte lover or an espresso enthusiast, coffee promises a rich journey with every cup."}' -H 'Content-Type: application/json' $(ENDPOINT_URL)/content/create
-	curl -X POST -d '{"content_text":"Venture beyond our blue planet and into the vast expanse of the universe with astronomy. This ancient science invites us to ponder the mysteries of stars, galaxies, and black holes. With telescopes as our windows, we can explore cosmic wonders, witness celestial events, and even trace our own place within this grand cosmic tapestry."}' -H 'Content-Type: application/json' $(ENDPOINT_URL)/content/create
-	curl -X POST -d '{"content_text":"Yoga, an ancient practice rooted in Indian philosophy, harmonizes the body, mind, and spirit. Through a combination of postures, breathing exercises, and meditation, yoga offers a holistic approach to wellness. Whether youre seeking physical flexibility, mental clarity, or spiritual growth, yoga provides a path to self-discovery and inner peace."}' -H 'Content-Type: application/json' $(ENDPOINT_URL)/content/create
-	curl -X POST -d '{"content_text":"In the intersection of technology and creativity lies digital art, a modern medium that pushes the boundaries of traditional artistry. With tools like graphic tablets and software, artists can create mesmerizing visuals, intricate designs, and immersive animations. Digital art not only showcases the evolution of art but also the limitless potential of human imagination when paired with technology."}' -H 'Content-Type: application/json' $(ENDPOINT_URL)/content/create
+	curl -X POST -d '{"content_title": "Come camping", "content_text":"Embrace the call of the wild with camping, an age-old tradition that reconnects us with nature. Whether youre pitching a tent under a canopy of stars or sharing stories around a crackling campfire, camping offers a unique escape from the hustle and bustle of daily life. Experience the tranquility of the great outdoors and create lasting memories with loved ones."}' -H 'Content-Type: application/json' $(ENDPOINT_URL)/content/create
```

Comment:
I don't think these work anyway since no token is being passed in the header.

Reply:
Yeah but thought I'd add anyway once we get back to fixing this